app/lib/pdf_generator.dart

```diff
@@ -21,58 +26,28 @@ class PDFGenerator {
             children: [
               pdf.Text(
                 "Pesticide Record Form",
-                style: pdf.TextStyle(
-                  fontSize: 24,
-                  fontWeight: pdf.FontWeight.bold,
-                ),
-              ),
-              pdf.SizedBox(height: 20),
-
-              pdf.Text(
-                "1. Applicator Name and License Number: ${applicator?.name} - ${applicator?.licenseNumber ?? 'N/A'}",
-              ),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text("2. Name and Address of Owner/Advisor: N/A"),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text(
-                "3. Owner Contacted: ${recordData.customerInformedOfRei ? 'Yes' : 'No'}",
-              ),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text("4. Crop Treated: N/A"),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text(
-                "5. Name and Coordinates of Field: ${recordData.fieldName}",
+                style: pdf.TextStyle(fontSize: 26, fontWeight: pdf.FontWeight.bold),
               ),
               pdf.SizedBox(height: 10),
 
-              pdf.Text("6. Pesticide Name and Registration Number/Rate: N/A"),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text("7. Total Treated Area: N/A"),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text("8. GPA: N/A"),
-              pdf.SizedBox(height: 10),
+              _buildRowWithBottomBox("Applicator Name and License Number", "${applicator?.name} - ${applicator?.licenseNumber}"),
+              _buildRowWithBottomBox("Name and Address of Owner/Advisor",
+                  "${customer?.name}, ${customer?.streetAddress}, ${customer?.city}, ${customer?.state}, ${customer?.zipCode}"),
+
+              _buildInlineRow("Owner Contacted", recordData.customerInformedOfRei ? 'Yes' : 'No'),
+              _buildInlineRow("Crop Treated", ""),
+              _buildInlineRow("Name and Coordinates of Field", recordData.fieldName),
+              _buildRowWithBottomBox("Pesticide Name and Registration Number/Rate", "", height: 50),

```

Comment:
```suggestion
              _buildRowWithBottomBox("Pesticides", "", height: 50),
```

I also think we will need to make this one taller if it needs to fit up to 10 pesticides. 

Maybe we could open up more vertical space by combining some of the environmental condition data into less lines?